requests-and-responses/requests-and-responses.md

```diff
@@ -36,6 +36,13 @@ Understand the concepts used in web development: HTTP requests + responses, HTML
 - [How DNS Works](https://howdns.works/ep1/)
```

Comment:
Are all of these really assigned in the outline? Please double check.

Also, could you put a header before the last 4 lines of this outline for Supplemental Materials?

Reply:
Yes, all of these are assigned in the outline.
I have added the Supplemental Materials